sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -455,16 +454,16 @@ public void recognizePiiEntitiesTooManyDocuments(HttpClient httpClient, TextAnal
                 }));
     }
 
-    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
-    @MethodSource("com.azure.ai.textanalytics.TestUtils#getTestParameters")
-    public void recognizePiiEntitiesWithDomain(HttpClient httpClient, TextAnalyticsServiceVersion serviceVersion) {
-        client = getTextAnalyticsAsyncClient(httpClient, serviceVersion);
-        recognizePiiEntitiesWithDomainRunner((inputs, options) ->
-            StepVerifier.create(client.recognizePiiEntitiesBatch(inputs, null, options))
-                .assertNext(response -> validatePiiEntitiesResultCollection(true,
-                    getExpectedBatchPiiEntitiesWithDomainSpecified(), response))
-                .verifyComplete());
-    }
+//    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
```

Comment:
instead of comment out I prefer deleting... better to introduce than to leave dead code in case the service changes the parameter